packages/vue/src/components/organisms/SfTable/_internal/SfTableHeading.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <tr class="sf-table__heading">
+    <!-- @slot Custom content for table heading -->
```

Comment:
@justyna-13 when we use default `<slot/> ` probably we want set main content, not custom :) what do you think?